### DISCUSSION TITLE
chatgpt API 参数

### DISCUSSION THREAD

**Discussion by HsunGong** [ChatGPTNextWeb/NextChat#791] [Ideas]
想让gpu-3.5-turbo表现的更像网页版的chatgpt
1，到底该怎么设定参数？
2，到底怎么评价相似度？

欢迎大家讨论

相关问题我也正在知乎上提问：

https://www.zhihu.com/question/595469072

相关问题也在Reddit上找到：

https://www.reddit.com/r/ChatGPT/comments/zvjdbi/what_api_parameters_are_used_in_the_current/

https://www.reddit.com/r/OpenAI/comments/10e9gfa/api_which_model_parameters_should_i_use_to_get/

https://www.reddit.com/r/OpenAI/comments/1101kpd/whats_currently_the_bestmost_promising_equivalent/

https://www.reddit.com/r/ChatGPT/comments/11gyfnj/comparison_of_chatgpt_webapp_and_gpt35turbo_api/

这个给了一个推荐，但是我感觉不太好用。。。

https://www.reddit.com/r/ChatGPT/comments/126sr15/gpt_api_analyzing_which_temperature_and_top_p/

这个有些分析

https://medium.com/@bezbos./openai-completions-api-complete-guide-8da45b3d780b

**Comment by Corwin006** [comment]
我没看你的链接，如果网上搜不到答案。
那只能猜测说，OpenAI提供的接口是ChatGPT的缩减版。
想想也是的，开放出来，让别人做个ChatGPT的竞品出来吗？
这个从产品战略角度也不会这么做。
另外你看API支持的模型版本比ChatGPT PLUS延后，等等策略，这都是护城河啊。
以上纯属个人观点，如果有不对之处请指正。
